src/com/epam/Member.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.training;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+public class Member {
+    public Scanner input = new Scanner(System.in);
+    public Scanner input1 = new Scanner(System.in);
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
```

Comment:
it is good practice to have formatter can be static private and final , and mane it with some meaningful name